Makefile

```diff
@@ -151,16 +151,21 @@ clean-glide-cache:
 deps: prebuild-check
 	$(GLIDE_BIN) --verbose install
 
-.PHONY: generate
-## Generate GOA sources. Only necessary after clean of if changed `design` folder.
-generate: prebuild-check $(DESIGNS) $(GOAGEN_BIN) $(GO_BINDATA_ASSETFS_BIN) $(GO_BINDATA_BIN)
+app/controllers.go: $(DESIGNS) $(GOAGEN_BIN)
```

Comment:
@sbose78 those are files that are generated and therefore the target is the name of the file.